tests/testthat/_snaps/backend-sqlite.md

```diff
@@ -22,10 +22,10 @@
       LEFT JOIN `df` AS `RHS`
       ON (`LHS`.`x` = `RHS`.`x`)
       UNION
-      SELECT COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`, `LHS`.`y` AS `y.x`, `RHS`.`y` AS `y.y`, `z`
+      SELECT COALESCE(`RHS`.`x`, `LHS`.`x`) AS `x`, `LHS`.`y` AS `y.x`, `RHS`.`y` AS `y.y`, `z`
```

Comment:
Should this have changed?

Reply:
This does not really matter because a) they are equal (see the `ON`) and b) the `COALESCE()` is not really needed here but rather an artifact from simulating the `full_join()`. As the result should be correct it should not matter that the query is a bit more complicated than necessary.

Reply:
I'm just slightly worried that this indicates we're now preferring the RHS of the join over the LHS in full joins?

Reply:
Hmm, I understand your concern but this was also the case before but just hidden because lazy frames are always named `df`. Before the second join query was actually

```sql
  SELECT
    COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`,
    `LHS`.`y` AS `y.x`,
    `z1`,
    `RHS`.`y` AS `y.y`,
    `z2`
  FROM `df2` AS `LHS`
  LEFT JOIN `df1` AS `RHS`
    ON (`LHS`.`x` = `RHS`.`x`)
```

So, `df2` was `LHS` instead of `RHS`. Now this is

```sql
SELECT
    COALESCE(`RHS`.`x`, `LHS`.`x`) AS `x`,
    `LHS`.`y` AS `y.x`,
    `z1`,
    `RHS`.`y` AS `y.y`,
    `z2`
  FROM `df2` AS `RHS`
  LEFT JOIN `df1` AS `LHS`
    ON (`RHS`.`x` = `LHS`.`x`)
```

Changing the `COALESCE` would basically require an extra argument to `sql_join_vars()` or writing a custom left join query method for SQLite. I don't think this is worth it...